pages/comunidade/index.js

```diff
@@ -95,7 +95,7 @@ const Community = ({ credentials }) => {
 Community.getInitialProps = async (ctx) => {
   if (!ctx.query.slug) {
     ctx.res.writeHead(302, {
-      Location: '/',
+      Location: '_error',
```

Comment:
@thejullius, acredito que você poderá ajudar mais com isso aqui. Quando fiz o teste entrando em /dashaboard, ele fica recarregando a página infinitamente. Então não sei qual seria a forma correta de fazer isso aqui.

Reply:
```suggestion
      Location: '/',
```
Aqui e nos outros casos, a gente ta travando 400 ou 401. Pro erro 404 a gente so precisa criar o component _error.js que o next resolve o resto